rotkehlchen/history/events.py

```diff
@@ -132,7 +144,7 @@ def get_history(
         """Creates trades and loans history from start_ts to end_ts"""
         self._reset_variables()
         step = 0
-        total_steps = len(self.exchange_manager.connected_exchanges) + HISTORY_QUERY_STEPS
+        total_steps = len(self.exchange_manager.connected_exchanges) + NO_HISTORY_QUERY_STEPS_EXCL_EXCHANGES  # noqa: E501
```

Comment:
I will fix in another PR or something, or you can do too. The abreviation for number we use in the codebase is `num`, not `no`.

No need for another commit in this PR for such a small thing.

Reply:
cool